trunk/src/app/srs_app_source.cpp

```diff
@@ -910,6 +910,15 @@ bool SrsOriginHub::active()
     return is_active;
 }
 
+srs_utime_t SrsOriginHub::cleanup_delay()
+{
+    srs_utime_t delay = hls->cleanup_delay();
+    if (delay > 0) {
+        return delay;
+    }
+    return dash->cleanup_delay();
```

Comment:
why the `cleanup_delay` of `SrsOriginHub` is the max of `hls` and `dash`?
```suggestion
    return std::max(dash->cleanup_delay(), hls->cleanup_delay);
```

Another puzzle, the default cleanup_delay of `hls` is 120 * 1.1 seconds = 132 seconds ~ 2 mins, while the `dash`'s cleanup_delay is 0 seconds, is that normal?

Reply:
Fixed. Note that we use `srs_max` instead.